Classes/WPEditorView.m

```diff
@@ -275,8 +275,17 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
             [newValue getValue:&newSize];
```

Comment:
Nice call!  That method is a 100% less prone to making coding mistakes.  Took me a while to find it though and got me scratching my head (because of the slightly different name, mind you, heh).